tests/browser/pe/program_enrollments.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+# Create A New Course
```

Comment:
Do we need this commented code for creating a new course ? If it is required for now or in the future, can we move it to HEDA.robot file ?

Reply:
it's not ready for HEDA.robot file.